components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_monitoring_schedule_component.py

```diff
@@ -156,54 +193,82 @@ def test_v2_monitoring_schedule_update(
+        monitoring_schedule_describe = ack_utils._get_resource(
+            k8s_client, monitoring_schedule_name, "monitoringschedules"
         )
-
-        # Check if the job definition is created
         assert (
-            job_definition_1_describe_ack["status"]["ackResourceMetadata"]["arn"]
-            != None
-        )
-
-        # Verify if monitoring schedule is created with correct name and endpoint
-        monitoring_schedule_describe = sagemaker_utils.describe_monitoring_schedule(
-            sagemaker_client, monitoring_schedule_name
+            monitoring_schedule_name
+            in monitoring_schedule_describe["status"]["ackResourceMetadata"]["arn"]
         )
         assert (
-            monitoring_schedule_name
-            in monitoring_schedule_describe["MonitoringScheduleArn"]
+            monitoring_schedule_describe["status"]["monitoringScheduleStatus"]
+            == "Scheduled"
         )
-
-        assert monitoring_schedule_describe["MonitoringScheduleStatus"] == "Scheduled"
-
-        assert monitoring_schedule_describe["EndpointName"] == deploy_endpoint
-
         assert (
-            monitoring_schedule_describe["MonitoringScheduleConfig"][
-                "MonitoringJobDefinitionName"
+            monitoring_schedule_describe["spec"]["monitoringScheduleConfig"][
+                "monitoringJobDefinitionName"
             ]
             == job_definition_name_1
         )
 
-        # Verify if job definition is created with correct instance type
-        job_definition_1_describe = (
-            sagemaker_utils.describe_data_quality_job_definition(
-                sagemaker_client, job_definition_name_1
-            )
+        # Verify if the job definition CR is created properly
+        job_definition_1_describe = ack_utils._get_resource(
+            k8s_client, job_definition_name_1, "dataqualityjobdefinitions"
+        )
+        assert (
+            job_definition_name_1
+            in job_definition_1_describe["status"]["ackResourceMetadata"]["arn"]
         )
         assert (
-            job_definition_1_describe["JobResources"]["ClusterConfig"]["InstanceType"]
+            job_definition_1_describe["spec"]["dataQualityJobInput"]["endpointInput"][
+                "endpointName"
+            ]
+            == deploy_endpoint
+        )
+        assert (
+            job_definition_1_describe["spec"]["jobResources"]["clusterConfig"][
+                "instanceType"
+            ]
             == "ml.m5.large"
         )
 
+        # Verify component output
+        outputs = {
+            "sagemaker-monitoringschedule": [
+                "ack_resource_metadata",
+                "monitoring_schedule_status",
+                "sagemaker_resource_name",
+            ]
+        }
+
+        output_files = minio_utils.artifact_download_iterator(
+            workflow_json, outputs, download_dir
+        )
+
+        output_ack_resource_metadata = json.loads(
```

Comment:
lets move ack_resource_meta extraction to a common method because its repeated in multiple tests

can be used by hosting tests too